web/themes/joinup/joinup_theme.theme

```diff
@@ -39,24 +42,35 @@ function joinup_theme_preprocess_page(&$variables) {
   // Set page title.
   $title = \Drupal::service('title_resolver')->getTitle($request, $route_match->getRouteObject());
 
+  // Check if page is form page and page content has no white background.
+  $is_form_container = FALSE;
+
   // We need to show the header in a blue variant on certain routes:
   // node and rdf entity forms, user forms.
   $route_name = $route_match->getRouteName();
   $blue_header_routes = [
+    'adms_validator.adms_validator_form',
     'collection.propose_form',
+    'contact_form.contact_page',
     'entity.node.edit_form',
     'entity.rdf_entity.edit_form',
-    'node.add',
-    'node.add_page',
     'rdf_entity.propose_form',
-    'rdf_entity.rdf_add',
```

Comment:
I was testing some more generic routes like this.
For example, the header is not there on:
- `/rdf_entity/add/asset_distribution`
- `rdf_entity/add/asset_release`

and others coming from `rdf_entity/add`. I know content is not meant to be added from these forms, yet the ticket is saying "all forms", so I don't see why not have the header on these as well, especially given that, for example, "Add RDF" button on a collection will lead here.